src/js/examples/es8-async-await.js

```diff
@@ -18,27 +18,36 @@ function napTime(duration) {
   });
 }
 
+
 /* 
-  Await requires a Promise to be returned and
-  can only be used inside an async function.
-*/ 
+  Await requires a Promise to be returned which we can await 
```

Comment:
This sentence doesn't really flow very well. Please rewrite it for clarity. Additionally I believe await can be used at a top level now rather than only in an async functions.